### DISCUSSION TITLE
Switch Styling

### DISCUSSION THREAD

**Discussion by FutureFutureTo** [livewire/flux#895] [Feature Requests] [answered]
Hey there, I can't find where to style the switch component. I can find it in the flux published components. Any ideas?

**Comment by jeffchown** [comment, marked as answer]
@FutureFutureTo You can publish the switch component using `php artisan flux:publish` then by choosing Switch from the menu.
After that, it will be located here: `resources/views/flux/switch.blade.php`

(The stub is located here: `vendor/livewire/flux-pro/stubs/resources/views/flux/switch.blade.php`)

**Comment by FutureFutureTo** [comment]
thank you - found it. appreciate it!

**Reply by joshhanley** [reply]
@FutureFutureTo glad you got it sorted! In future please don't open a feature request for questions (as they're not really feature requests), open an issue instead 🙂
